.github/workflows/pypi_release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         TWINE_PASSWORD: ${{ secrets. PYPI_PASSWORD }}
         RELEASE: 1
       run: |
-        for v in common core features tabular multimodal vision timeseries autogluon
+        for v in common core features tabular multimodal timeseries autogluon
```

Comment:
Is the intention to not release autogluon.vision 0.7? I remember we talked about two options so I'd just like to double check: 

1. Don't release autogluon.vision 0.7.
2. Release autogluon.vision 0.7, but don't have it as part of the default autogluon install.

Based on the code changes here, this will not release autogluon.vision 0.7.

Reply:
Note: I am fully supportive of either option

Reply:
I'm supportive in just not releasing vision 0.7.

Reply:
Given `text` is already removed from pypi, I think `vision` can be removed too. If user wants to stick with `ag.vision`, they can `pip install autogluon.vision` from any previous version. So maybe it is easier just not release vision in v0.7.